### ISSUE TITLE
Change from "master" to "main"

### ISSUE THREAD

**Issue by jakefreeberg** [launchdarkly/node-server-sdk#237]
I'd assume you've heard the reasons... As a customer I'd just feel better if your projects were on the main branch.

https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main

**Comment by eli-darkly** [comment]
Yes, it's an ongoing process with many projects. We've already done it in some of our internal development repos. There's a large number of SDK projects, and to avoid unexpectedly disrupting developers who have cloned these projects, we want to approach it in a coordinated way. Also, it's not specific to `node-server-sdk`, so I'd like to close this issue while acknowledging the intent.
